code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -186,13 +186,15 @@
 	..()
 
 /mob/living/carbon/human/proc/Stasis(amount)
-	if((species.flags & NO_SCAN) || isSynthetic())
+	//if((species.flags & NO_SCAN) || isSynthetic()) vorestation removal
+	if(((species.flags & NO_SCAN) && !istype(species, /datum/species/xenochimera)) || isSynthetic()) //vorestation addition

```

Comment:
Okay, looking over it (and testing), transforming really really really fucks up too much stuff. 
HOWEVER turns out the does_not_breathe var does exactly what I wanted out of stasis. It's used by the self-respiration changeling power.
Yes, chimera can bleed out and die while regenerating if they have wounds/IB. I see no reason to make this a thing they can avoid by taking a nap.